clippy_lints/src/utils/conf.rs

```diff
@@ -156,6 +156,8 @@ define_Conf! {
     (verbose_bit_mask_threshold, "verbose_bit_mask_threshold", 1 => u64),
     /// Lint: DECIMAL_LITERAL_REPRESENTATION. The lower bound for linting decimal literals
     (literal_representation_threshold, "literal_representation_threshold", 16384 => u64),
+    /// Lint: TRIVIALLY_COPY_PASS_BY_REF. The maximum size to consider a copy type as "trivial".
+    (trivial_copy_size_limit, "trivial_copy_size_limit", None => Option<u64>),
```

Comment:
Oh this configuration is in bytes not in bits. I thought it was in bits for no good reason 😃 . Could you add `(in bytes)` or something similar to the documentation?